cheroot/wsgi.py

```diff
@@ -254,6 +255,9 @@ def get_environ(self):
             # AF_UNIX. This isn't really allowed by WSGI, which doesn't
             # address unix domain sockets. But it's better than nothing.
             env['SERVER_PORT'] = ''
+            uid = self.req.conn.get_peer_uid()
+            if uid is not None:
+                env['_REMOTE_UID'] = str(uid)
```

Comment:
1) This shouldn't be a default behavior to avoid overhead for other users who may not want such logic to be triggered, so I would prefer that the calling logic go to CherryPy tool, which would be easy to turn on or switch off easily via config option.
2) It would be enough to provide `@properties`, which evaluate creads lookup lazily, once per request as I commented earlier.
3) It would be ok to have some function/method populating all of `REMOTE_USER`, `X_REMOTE_UID`, `X_REMOTE_GID`, `X_REMOTE_PID` or just some of them as configured.